pulsar-broker/src/main/java/org/apache/pulsar/PulsarClusterMetadataSetup.java

```diff
@@ -464,9 +476,12 @@ static void createPartitionedTopic(MetadataStore configStore, TopicName topicNam
         }
     }
 
-    public static MetadataStoreExtended initLocalMetadataStore(String connection, int sessionTimeout) throws Exception {
```

Comment:
Is it useful to keep backwards compatibility by keeping this method signature and delegating to the new method signature with `configPath` set to `null`? Just thinking of possible cases where this class is used by some external code.

Reply:
Good point, updated.